src/style.css

```
@@ -0,0 +1,84 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 700;
+}
+
+html {
+  font-family: "Avenir" , sans-serif;
+  color: #3d4e61;
+  background-color: #e5e5e5;
+}
+
+.search_field {
+  position: relative;
+  margin: 20px auto;
```

```suggestion
  margin-top: 20px;
```
I think, this horizontal margin is the reason why your tests have failed